components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1106,6 +1111,20 @@ void RewardsServiceImpl::LoadActivityInfo(
                      filter.id));
 }
 
+void RewardsServiceImpl::OnPublisherActivityInfoLoaded(
+    ledger::PublisherInfoCallback callback,
+    uint32_t result,
+    const std::string& info_json) {
+  std::unique_ptr<ledger::PublisherInfo> publisher;
+  publisher.reset(new ledger::PublisherInfo());
+
+  if (!info_json.empty()) {
+    publisher->loadFromJson(info_json);
+  }
+
+  callback(static_cast<ledger::Result>(result), std::move(publisher));
```

Comment:
From above, the bat_ledger callback returns `ledger::Result` to implicit uint32 whereas `ledger::Result unless there's a change I'm unaware of.

Reply:
left comment above